java-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java

```diff
@@ -7,28 +7,21 @@
  */
 public interface HumanProfile extends Profile {
     /**
-     * Corresponds to primary name in passport, and surname in English. 
+     * Corresponds to primary name in passport, and surname in English.
      * @return the surname
      */
     String getFamilyName();
 
     /**
      * Corresponds to secondary names in passport, and first/middle names in English.
-     * 
+     *
      * @return the name
      */
     String getGivenNames();
 
     /**
-     * Equal to ${given_names} + " " + ${family_name}.
```

Comment:
Done already, this commit was reverted by a later commit and the method was deprecated. This change should show as "outdated" in the github UI since it was superseded by a later commit that reverted this change.